app/api_gateway.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import FastAPI, HTTPException
+import httpx
+
+app = FastAPI()
+MICROSERVICE_URL = "http://localhost:8001"
+
+@app.get("/home")
+async def get_home_from_microservice():
+    async with httpx.AsyncClient() as client:
+        try:
+            response = await client.get(f"{MICROSERVICE_URL}/home")
```

Comment:
to w przyszłości będzie odpytywało endpointy z innych mikroserwisów, tj. innych aplikacji